xrviz/dashboard.py

```diff
@@ -1,7 +1,9 @@
 import panel as pn
 import xarray as xr
 import hvplot.xarray
-from cartopy import crs as ccrs
+import holoviews as hv
+from cartopy import crs
```

Comment:
Maybe the import should be deferred until it is needed

Reply:
Corrected. Thanks!